cli.js

```diff
@@ -75,63 +77,53 @@ bionode-ncbi plink tax sra -s --pretty`
 .boolean('pretty')
 .describe('pretty', 'Print human readable output instead of NDJSON')
 .choices('dlsource', ['assembly', 'sra'])
-.choices('db', [
-  'gquery', 'assembly', 'bioproject', 'biosample', 'biosystems', 'books',
-  'clinvar', 'clone', 'cdd', 'gap', 'dbvar', 'nucest', 'gene', 'genome', 'gds',
-  'geoprofiles', 'nucgss', 'gtr', 'homologene', 'medgen', 'mesh', 'ncbisearch',
-  'nlmcatalog', 'nuccore', 'omim', 'pmc', 'popset', 'probe', 'protein',
-  'proteinclusters', 'pcassay', 'pccompound', 'pcsubstance', 'pubmed',
-  'pubmedhealth', 'snp', 'sparcle', 'sra', 'structure', 'taxonomy', 'toolkit',
-  'toolkitall', 'toolkitbook', 'toolkitbookgh', 'unigene'
-])
-.example('databases available',
-`gquery (All Databases), assembly, bioproject, biosample, biosystems, \
-books, clinvar, clone, cdd (Conserved Domains), gap (dbGaP), dbvar, \
-nucest (EST), gene, genome, gds (GEO DataSets), geoprofiles (GEO Profiles), \
-nucgss (GSS), gtr (GTR), homologene, medgen, mesh, \
-ncbisearch (NCBI Web Site), nlmcatalog, nuccore (Nucleotide), omim, pmc, \
-popset, probe, protein, proteinclusters, pcassay (PubChem BioAssay), \
-pccompound (PubChem Compound), pcsubstance (PubChem Substance), \
-pubmed, pubmedhealth, snp, sparcle, sra, structure, \
-taxonomy, toolkit, toolkitall, toolkitbook, toolkitbookgh, unigene`
-)
+.example('databases available', validDbs.printDbs())
```

Comment:
Please add `.choices('db', validDbs.printDbs())` before this line. Otherwise, you don't get an error if you choose a wrong database.